[2023-07-21T19:01:18.548+0530] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Data_processing.run_spark scheduled__2023-07-20T11:00:00+00:00 [queued]>
[2023-07-21T19:01:18.557+0530] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Data_processing.run_spark scheduled__2023-07-20T11:00:00+00:00 [queued]>
[2023-07-21T19:01:18.557+0530] {taskinstance.py:1308} INFO - Starting attempt 5 of 7
[2023-07-21T19:01:18.594+0530] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): run_spark> on 2023-07-20 11:00:00+00:00
[2023-07-21T19:01:18.599+0530] {standard_task_runner.py:57} INFO - Started process 268407 to run task
[2023-07-21T19:01:18.606+0530] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'Data_processing', 'run_spark', 'scheduled__2023-07-20T11:00:00+00:00', '--job-id', '470', '--raw', '--subdir', 'DAGS_FOLDER/assignment2.py', '--cfg-path', '/tmp/tmpirlkingj']
[2023-07-21T19:01:18.607+0530] {standard_task_runner.py:85} INFO - Job 470: Subtask run_spark
[2023-07-21T19:01:18.668+0530] {task_command.py:410} INFO - Running <TaskInstance: Data_processing.run_spark scheduled__2023-07-20T11:00:00+00:00 [running]> on host neosoft-Latitude-E5270
[2023-07-21T19:01:18.778+0530] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='nikitaneti21@gmail.com' AIRFLOW_CTX_DAG_OWNER='Nikita' AIRFLOW_CTX_DAG_ID='Data_processing' AIRFLOW_CTX_TASK_ID='run_spark' AIRFLOW_CTX_EXECUTION_DATE='2023-07-20T11:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='5' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-07-20T11:00:00+00:00'
[2023-07-21T19:01:18.780+0530] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-07-21T19:01:18.781+0530] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'spark_command']
[2023-07-21T19:01:18.788+0530] {subprocess.py:86} INFO - Output:
[2023-07-21T19:01:18.789+0530] {subprocess.py:93} INFO - /usr/bin/bash: spark_command: command not found
[2023-07-21T19:01:18.789+0530] {subprocess.py:97} INFO - Command exited with return code 127
[2023-07-21T19:01:18.799+0530] {taskinstance.py:1824} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/neosoft/airflow/py_env/lib/python3.8/site-packages/airflow/operators/bash.py", line 210, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 127.
[2023-07-21T19:01:18.803+0530] {taskinstance.py:1345} INFO - Marking task as UP_FOR_RETRY. dag_id=Data_processing, task_id=run_spark, execution_date=20230720T110000, start_date=20230721T133118, end_date=20230721T133118
[2023-07-21T19:01:18.829+0530] {base.py:73} INFO - Using connection ID 'smtp_default' for task execution.
[2023-07-21T19:01:18.830+0530] {email.py:268} INFO - Email alerting: attempt 1
[2023-07-21T19:01:20.295+0530] {email.py:280} INFO - Sent an alert email to ['nikitaneti21@gmail.com']
[2023-07-21T19:01:22.039+0530] {standard_task_runner.py:104} ERROR - Failed to execute job 470 for task run_spark (Bash command failed. The command returned a non-zero exit code 127.; 268407)
[2023-07-21T19:01:22.079+0530] {local_task_job_runner.py:225} INFO - Task exited with return code 1
[2023-07-21T19:01:22.145+0530] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
